<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUser</class>
 <widget class="QDialog" name="AddUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover:!pressed
{
  	border: 4px solid white;
	background-color: red;
}

QLineEdit :hover:!pressed
{
  	border: 4px solid white;
	background-color: red;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>561</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(255, 114, 116,210);
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>210</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Arial&quot;;
background-color: rgb(255, 119, 137,0);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_TenDangNhap_AddUser">
     <property name="text">
      <string>Tên đăng nhập</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_MaPin_AddUser">
     <property name="text">
      <string>Mã pin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_NhapLaiMaPin_AddUser">
     <property name="text">
      <string>Nhập lại mã pin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_HoVaTen_AddUser">
     <property name="text">
      <string>Họ và tên</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Email_AddUser">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CongViec_AddUser">
     <property name="text">
      <string>Công việc</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_TenDangNhap_AddUser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 7px;
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_MaPin_AddUser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 7px;
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NhapLaiMaPin_AddUser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 7px;
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_HoVaTen_AddUser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 7px;
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Email_AddUser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 7px;
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CongViec_AddUser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid gray;
border-radius: 7px;
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DangKy_AddUser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
color: rgb(255, 255, 255);
background-color: rgb(48, 165, 255);
font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Đăng ký</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/BackGr/bkgr_SignUp.jpg);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Images/BackGr/budget.jpg);
border: 2px solid yellow;
border-radius: 40px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_AddUser">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>911</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 26pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(183, 255, 245,0);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CHÀO MỪNG BẠN ĐẾN TRANG ĐĂNG KÝ ^^&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_AddUser</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
